README.md

```diff
@@ -0,0 +1,154 @@
+# uwv\_dynamic\_model: Implementation of a dynamic model for undewater vehicles
+
+Assumptions:
+
+ - A vehicle, considered a rigid body, is represented by a point in space, been this point the center of gravity (CoG).
+ - The CoG is the origin of the right-hand side body-frame.
+ - There is no representation of multiples objects, neither contact or interaction betwen them.
+ - There is no representation of the environment (floor, surface, water currents, waves, etc)[^TODO].
```

Comment:
Since Github does not accept this footnote style ([^TODO]), I'd suggest working it around as suggested in this link:

http://stackoverflow.com/questions/25579868/how-to-add-footnotes-to-github-flavoured-markdown

It's not ideal, but I think it's better than letting some [^] along the text

```
Expanding a little bit on the previous answer, you can make the footnote links clickable here as well. First define the footnote at the bottom like this

<a name="myfootnote1">1</a>: Footnote content goes here
Then reference it at some other place in the document like this

<sup>[1](#myfootnote1)</sup>
```